test/unit/Libify.js

```diff
@@ -0,0 +1,308 @@
+import React from 'react';
+import {xdr, Transaction, Operation} from 'stellar-sdk';
+import Libify from '../../src/utilities/Libify';
+
+describe('Libify-built', () => {
```

Comment:
IMO you should change the name of this to reflect the function being tested: `buildTransaction`